internal/bed/bed.go

```diff
@@ -0,0 +1,58 @@
+package bed
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// Note that the first column is 1 when user sets column locations,
+// but that .VerifyAndHandle() will correct this to standard indexes
+type Bedfile struct {
+	Input     string   `env:"INPUT_FILE" required:"" short:"i" help:"Bed file path"`
+	Output    string   `env:"OUTPUT_FILE" short:"o" help:"Path for the output file. If unset the output will be written to stdout"`
+	StrandCol int      `env:"STRAND_COL" help:"The column containing the strand information (first column is 1)"`
```

Comment:
Yes, exactly!
I found it a bit difficult to define what it should say. Most users would probably think that the counting of the column starts with 1 anyway? So maybe I'm overthinking this and we should actually just remove it?

Reply:
Regarding that strand and feature columns has to be > 3, I guess this is given by the bed file format itself?

Reply:
I think it is worth mentioning that the column index is one-based. Also, it probably helps to indicate that these additional columns are greater than 3 because not everyone might look up the exact bed format. What happens if strand and feature are not indicated (`0`)?

Reply:
I have now updated the help text to link to the bed file format standard that we are using: 8b3343cbabffe4bdb5666d0f0da4aa3c365617f4

And have updated the help text for `StrandCol` and `FeatCol` to indicate the 1-based indexing of the columns: 38ee4668c62678d0a4d6f986f11a7f4a19766f58

What do you think?

Regarding what happens if strand and feature is not indicated; they are simply not used 🙂